homeassistant/components/conversation/__init__.py

```diff
@@ -341,6 +341,14 @@ def async_get_agent_info(
     return None
 
 
+async def async_get_agent(
```

Comment:
We shouldn't add this. The create default pipeline can use `async_get_conversation_languages`

Reply:
OK, fixed 👍